src/front/js/store/flux.js

```diff
@@ -59,8 +65,14 @@ const getState = ({ getStore, getActions, setStore }) => {
 			},
 
 			getMessage: () => {
+				const store = getStore();
+				const opts = {
+					headers: {
+						"Authorization" : "Bearer " + store.token
+					}
+				}
 				// fetching data from the backend
-				fetch(process.env.BACKEND_URL + "/api/hello")
+				fetch("https://3001-nozigs-nozigs-8fn6hvofjfr.ws-eu34.gitpod.io/api/hello", opts)
```

Comment:
this, we will need to have it back in the future bc the URL will change when we deploy /!\